dev/com.ibm.ws.jaxrs.2.0.client_fat/fat/src/com/ibm/ws/jaxrs20/client/fat/test/JAXRS20ClientInvocationTest.java

```diff
@@ -69,6 +68,7 @@ public void afterTest() {
     }
 
     @Test
+    @SkipForRepeat("EE9_FEATURES") // Continue to skip this test for EE9 as the RestEasy ClientImpl is org.jboss.resteasy.client.jaxrs.internal.ResteasyClientImpl 
```

Comment:
Instead of skipping this, could you change the test to be something like:
```
if (JakartaEE9Action.isActive()) {
    this.runTestOnServer(invocationTarget, "testClientClass", p, "org.jboss.resteasy.client.jaxrs.internal.ResteasyClientImpl");
} else {
    this.runTestOnServer(invocationTarget, "testClientClass", p, "com.ibm.ws.jaxrs20.client.JAXRSClientImpl");
}
```

Reply:
Sure